lnwallet/channel_test.go

```diff
@@ -3624,6 +3804,23 @@ func TestChanSyncOweRevocation(t *testing.T) {
 
 		testChanSyncOweRevocation(t, taprootBits)
 	})
+	t.Run("taproot", func(t *testing.T) {
```

Comment:
Yeah this is a dup. Now that the dependent PR is in, I can rebase to get rid of that.

Reply:
Err rather, #9046 was merged into master, but I'll rebase this on top of that to include those commits, as we also need that in the staging branch since this is what `litd` points to.